service-loadbalancer/service_loadbalancer.go

```diff
@@ -367,6 +377,14 @@ func (lbc *loadBalancerController) getServices() (httpSvc []service, tcpSvc []se
 			glog.Infof("Ignoring service %v, it already has a loadbalancer", s.Name)
 			continue
 		}
+
+		if *nameLabel != "" {
```

Comment:
please store nameLabel in loadBalancerController like we do the other cmd line flags instead of accessing it as a global